systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterIT.java

```diff
@@ -382,4 +382,24 @@ public void testForTopicOperator() {
         List<String> topics = listTopicsUsingPodCLI(CLUSTER_NAME, kafkaPodName(CLUSTER_NAME, 1));
         assertThat(topics, not(hasItems("topic-from-cli", "my-topic")));
     }
-}
+
+    @Test
+    @JUnitGroup(name = "regression")
+    @KafkaCluster(name = CLUSTER_NAME,
+            kafkaNodes = 1,
+            zkNodes = 1,
+            config = {
+                    @CmData(key = "kafka-rack",
+                            value = "{\"topologyKey\": \"rack-key\"}")
+            })
+    public void testRackAware() {
```

Comment:
I think this should also test the config file and whether it contains the proper value.